### ISSUE TITLE
Groups with special Character can not be addressed

### ISSUE THREAD

**Issue by xbluemonkx** [nextcloud/spreed#11935]
## Steps to reproduce
1. Create a user group, that has a Special Charakter in it (Umlaute: äöü) (eg: "Bälle" - german word for "Balls")
2. Add this group to a chat
3. reference the group in a message using @group

### Expected behaviour
it should work like it works with users or groups without special character. The group should be converted into that visual icon of the group.

### Actual behaviour
Message looks like this: "Hello Friends of @"group/Bälle" can someone please call me?"

## Talk app
Talk 18.0.5
Nextcloud 28.0.3
Firefox

**Comment by nickvergessen** [comment]
Yeah the server code only supports a-zA-Z0-9 at the moment for the group IDs:
https://github.com/nextcloud/server/blob/e5b0c06b2449d9015ed1ce5d0fd027836babc939/lib/private/Comments/Comment.php#L223

However you could create a group `Baelle` with the displayname `Bälle`:
```
occ  group:add --help
Description:
  Add a group

Usage:
  group:add [options] [--] <groupid>

Arguments:
  groupid                          Group id

Options:
      --display-name=DISPLAY-NAME  Group name used in the web UI (can contain any characters)
```

At least on CLI the option is easily accessible.

In the web you would need to create the group as `Baelle` and then rename it afterwards.

**Comment by xbluemonkx** [comment]
Hi @nickvergessen,

thanks for the input. this bug is anoying, but not critical. therefore we will just live with it until there is a fix in the server code.

but the workaround looks good and I am glad you wrote it down. maybe someone else will use it.

**Comment by xbluemonkx** [comment]
@danxuliu @nickvergessen  - this bug is still present, but was added to the already closed version 19/patch 14. - is this still on the roadmap? can we re-assign it to a correct milestone?
